<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordWindow</class>
 <widget class="QMainWindow" name="PasswordWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="warning_1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>390</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : transparent;
color : rgb(117,247,88);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>121</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="Telephone">
       <property name="styleSheet">
        <string notr="true">font: 75 15pt &quot;华文行楷&quot;;
color : rgb(255,255,255);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;*&lt;/span&gt;Telephone:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Identify">
       <property name="styleSheet">
        <string notr="true">font: 75 15pt &quot;华文行楷&quot;;
color : rgb(255,255,255);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;*&lt;/span&gt;Identify：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>191</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="tephone">
       <property name="styleSheet">
        <string notr="true">background-color : transparent;
border-color: : transparent;
color : rgb(255,255,255);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Identify_e">
       <property name="styleSheet">
        <string notr="true">background-color : transparent;
border-color: : transparent;
color : rgb(255,255,255);</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="nextBtn">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>440</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
                   background-color:rgba(220,220,220,120);
                   border-style:outset;                                                             
                   border-radius:5px;                     
                   font: 75 15pt &quot;华文行楷&quot;;           
                   padding:6px;
                   }

                   QPushButton:pressed{
                   background-color:rgba(100,100,100,180);
                   border-color:rgba(100,100,100,30);
                   border-style:inset;
                   }
                   QPushButton:hover{
                   background-color:rgba(100,255,100,100);
                   border-color:rgba(255,255,255,200);
}
</string>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="okBtn">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>440</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
                   background-color:rgba(220,220,220,120);
                   border-style:outset;                                                             
                   border-radius:5px;                     
                   font: 75 15pt &quot;华文行楷&quot;;           
                   padding:6px;
                   }

                   QPushButton:pressed{
                   background-color:rgba(100,100,100,180);
                   border-color:rgba(100,100,100,30);
                   border-style:inset;
                   }
                   QPushButton:hover{
                   background-color:rgba(100,255,100,100);
                   border-color:rgba(255,255,255,200);
}
</string>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>124</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="Newpasswd">
       <property name="styleSheet">
        <string notr="true">font: 75 15pt &quot;华文行楷&quot;;
color : rgb(255,255,255);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Newpassword:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Confirm">
       <property name="styleSheet">
        <string notr="true">font: 75 15pt &quot;华文行楷&quot;;
color : rgb(255,255,255);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Confirm：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>191</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="newPasswd">
       <property name="styleSheet">
        <string notr="true">background-color : transparent;
border-color: : transparent;
color : rgb(255,255,255);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="confirm">
       <property name="styleSheet">
        <string notr="true">background-color : transparent;
border-color: : transparent;
color : rgb(255,255,255);</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="page2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>441</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(64,66,68,120);

border-radius:8px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="warning_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>390</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : transparent;
color : rgb(117,247,88);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="backBtn">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>450</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
                   background-color:transparent;
                   border-style:outset;                                                             
                   border-radius:5px;                     
                   font: 75 15pt &quot;华文行楷&quot;;           
                   padding:6px;
                   }

                   QPushButton:pressed{
                   background-color:rgba(100,100,100,180);
                   border-color:rgba(100,100,100,30);
                   border-style:inset;
                   }
                   QPushButton:hover{
                   background-color:transparent;
                   color:rgba(255,255,255,200);
}
</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>360</y>
      <width>191</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
    border: 1px solid white;   
    border-radius: 3px;   
    padding: 1px 18px 1px 3px;   
    color: #fff;
    font: normal normal 15px &quot;Microsoft YaHei&quot;;
    background: transparent;
}
/* 下拉后，整个下拉窗体每项的样式 */
QComboBox QAbstractItemView::item {
    height: 30px;   /* 项的高度（设置pComboBox-&gt;setView(new QListView());后，该项才起作用） */
	background-color: rgba(100,106,100,150);
	color: rgb(64,64,64);
}
QComboBox QAbstractItemView::item:hover{
	background-color: rgba(255,255,191,100);
	color: rgb(64,64,64);
}
/* QComboBox中的垂直滚动条 */
QComboBox QAbstractScrollArea QScrollBar:vertical {
    width: 10px;
    background-color: #d0d2d4;   /* 空白区域的背景色  灰色green */
	border-radius: 3px;   
}


</string>
    </property>
   </widget>
   <zorder>page2</zorder>
   <zorder>warning_1</zorder>
   <zorder>verticalLayoutWidget_2</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>nextBtn</zorder>
   <zorder>okBtn</zorder>
   <zorder>verticalLayoutWidget_3</zorder>
   <zorder>verticalLayoutWidget_4</zorder>
   <zorder>warning_2</zorder>
   <zorder>backBtn</zorder>
   <zorder>comboBox</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>690</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
